src/inverse-kinematics/include/iDynTree/InverseKinematics.h

```diff
@@ -378,6 +394,18 @@ class iDynTree::InverseKinematics
      *
      * @note This method returns true even if the frame constraint was already deactivated, it only
      *       returns false if the constraint was never added.
+<<<<<<< HEAD
+     *
+     * @param frameName       the name of the frame on which to attach the constraint
+     * @return true if successful (i.e. the constraint is present) , false otherwise.
+     */
+    bool deactivateFrameConstraint(const std::string& frameName);
+
+    /*!
+     * Check if a given constraint is active or not.
+
+     *
+=======
```

Comment:
I am afraid there is some unresolved conflicts @hu-yue

Reply:
Sorry! Did it too much in a hurry I will solve it tomorrow.